torcharrow/velox_rt/numerical_column_cpu.py

```diff
@@ -895,25 +871,6 @@ def is_monotonic_decreasing(self):
                 prev = current
         return True
 
-    # interop ----------------------------------------------------------------
-
-    @trace
-    def to_pandas(self):
-        """Convert self to pandas dataframe"""
-        # TODO Add type translation
-        # Skipping analyzing 'pandas': found module but no type hints or library stubs
-        import pandas as pd  # type: ignore
-
-        return pd.Series(self._ma())
```

Comment:
`._ma()` doesn't exists in cpu column (it's the numpy masked array). So this causes `to_arrow/to_pandas` failing on cpu column. 

Remove them for now so it use the default `to_array()` implementation by Python iterable.